### PR TITLE
Update `on_success` docstring

### PR DESCRIPTION
**Pull request by Jack-Keene**

### CODE REVIEW

File: snowplow_tracker/emitter_configuration.py [hunk 1]
```diff
@@ -39,7 +39,9 @@ def __init__(
         :param batch_size:     The maximum number of queued events before the buffer is flushed. Default is 10.
         :type  batch_size:     int | None
         :param on_success:      Callback executed after every HTTP request in a flush has status code 200
-                                Gets passed the number of events flushed.
+                                Gets passed one argument:
+                                1) If method is "post": The sent data in string form;
+                                   If method is "get":  An array of dictionaries corresponding to the sent events' payloads
```
Comment (matus-tomlein) [inline snowplow_tracker/emitter_configuration.py:44]: Nit: missing the 2) numbering in all modified docstrings.

```suggestion
                                 2) If method is "get":  An array of dictionaries corresponding to the sent events' payloads
```

File: snowplow_tracker/emitters.py [hunk 2]
```diff
@@ -83,7 +83,9 @@ def __init__(
         :param batch_size:  The maximum number of queued events before the buffer is flushed. Default is 10.
         :type  batch_size:  int | None
         :param on_success:  Callback executed after every HTTP request in a flush has status code 200
-                            Gets passed the number of events flushed.
+                            Gets passed one argument:
+                            1) If method is "post": The sent data in string form;
```
Comment (matus-tomlein) [inline snowplow_tracker/emitters.py:87]: Isn't it the other way? The post method is passed a list of payload dictionaries and the get method is passed the request data as string? Also if this is the case, we might need to update the type hint for the callback as currently it only covers the list of dictionaries.

**Review by matus-tomlein — APPROVED** [review]
LGTM!

**Review by greg-el — APPROVED** [review]
Wonderful work, Jack.

**Pull request merged**